mbed_app.json

```diff
@@ -507,8 +500,9 @@
             "storage.storage_type"                      : "TDB_INTERNAL"
         },
         "UHURU_RAVEN": {
+            "target.features_add"                       : ["BOOTLOADER", "STORAGE"],
```

Comment:
"update-client.storage-locations"           : 1,
            
            missing here?

Reply:
Already in place.